### PR TITLE
Edge Example App: Sense the Internet Part 2

### PR DESCRIPTION
**Pull request by ahmadateya**

### CODE REVIEW

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+This app is entirely depends on the backend app, so you need to deploy the backend app first, then you can deploy this app.
+
+All you need have to use this app is to have S3 bucket to store the data, and to configure the backend app to use this bucket.
+
+## Future work Ideas
+- Add more filters to the Network Topology graph (e.g. by date etc.).
+- Add more information to the devices list (e.g. device type, device location etc.).
+
+## GitHub Repositories
+- [https://github.com/ahmadateya/flotta-edge-example](https://github.com/ahmadateya/flotta-edge-example) <= Device App
```
Comment (pkliczewski) [inline _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:48]: Let's transfer those repos to project-flotta organization.

Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:48]: ok, How to do so?

**Review by pkliczewski — APPROVED** [review]

**Comment by pkliczewski** [comment]
@gciavarrini what do you think about this PR?

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- gsoc
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has two main features:
+- Sensing the Internet (which helps to construct devices network topology). <= this article (Part 2)
+- Read CPU temperature (which indicate how much load the device is handling).
+
+## How this app works
+This app is the presenting layer of the Flotta Edge devices, get its data from the backend app endpoints and display it to the user.
```
Comment (gciavarrini) [inline _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:20]: WDYT to add link to the backend app repo?

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+you can list all devices that uses your S3 Bucket as storage bucket, and for each device you can see its information and its network topology.
```
Comment (gciavarrini) [inline _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:22]: s/you/You/

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Device page with app features
+
+<img src="/assets/images/device-features-page.png" alt="List devices" width="800"/>
+
+- Network topology page
+
+<img src="/assets/images/network-topology-page.png" alt="List devices" width="800"/>
+
+## How to use this app
+This app is entirely depends on the backend app, so you need to deploy the backend app first, then you can deploy this app.
```
Comment (gciavarrini) [inline _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:39]: Can you add some info or a link on how to deploy the backed app?

In addition, "this app" appears two time in the sentence: whad do you think to reword it a bit?

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+All you need have to use this app is to have S3 bucket to store the data, and to configure the backend app to use this bucket.
+
+## Future work Ideas
```
Comment (gciavarrini) [inline _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown:43]: s/Idea/idea/

**Review by gciavarrini — CHANGES_REQUESTED** [review]

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown (hunk 2 above)
Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: good idea, added it

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown (hunk 3 above)
Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: done

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown (hunk 4 above)
Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: I can't think about an alternative for the two times `this app`, what could be a good replacement for it?

Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: I can't think about an alternative for the two times `this app`, what could be a good replacement for it?

Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: but I added more links to the sentence to deploying backend and frontend app too

Comment (gciavarrini) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: > I can't think about an alternative for the two times this app, what could be a good replacement for it?

WDYT, something like "The backend app is the only requirement, so deploy the backend app and then you're ready to deploy this app." (feel free to reword it!)

**Review by gciavarrini — APPROVED** [review]

**Comment by gciavarrini** [comment]
@ahmadateya can you check why `CheckLinks` is failing?

File: _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown (hunk 4 above)
Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown]: done

**Comment by ahmadateya** [comment]
I run `make check-links` before pushing the code and there are no mentions for other posts here!

**Comment by gciavarrini** [comment]
The log says

```
- ./flotta/2022/09/03/edge-example-app-sense-the-internet-part-1.html
  *  internally linking to /flotta/2022/09/05/edge-example-app-sense-the-internet-part-2.html, which does not exist (line 200)
     <a href="/flotta/2022/09/05/edge-example-app-sense-the-internet-part-2.html">part 2</a>
```

so it fails because of part1 link to part2 post.

Let's merge this PR.

**Pull request merged**